assets/scss/components/_hscroll.scss

```diff
@@ -59,15 +59,29 @@ Class                       | Description
 	}
 }
 
+.hscrollContainer, .hscroll {
+	margin-bottom: -#{$space-and-half};
+}
+
+.hscrollContainer {
+	overflow: hidden;
+}
+
 .hscroll {
 	-webkit-overflow-scrolling: touch;
 	-ms-overflow-style: -ms-autohiding-scrollbar;
 	overflow-x: scroll;
+	-ms-overflow-style: auto
+}
+
+.hscroll::-webkit-scrollbar {
+	display: none !important;
 }
 
 .hscroll-content {
 	@extend %inlineblockList;
 	box-sizing: content-box;
+	margin-bottom: $space-and-half !important; // !important to kill styles coming from `sassquatch.css`, being imported by `swarm-sasstools`
```

Comment:
There's a path to it on line 85 of `Gruntfile.js`, under the `preprocess` task. Not sure if this is compiling it and inlining it with the rest of the CSS or what.